tests/test_rest.py

```diff
@@ -443,5 +445,168 @@ def test_yaml(self):
         self.assertEqual(res.body, expected)
 
 
+class TestSchemaDecorator(unittest.TestCase):
+
+    def setUp(self):
+        self.root_app = bottle.default_app()
+        self.app = webtest.TestApp(self.root_app)
+
+        def callback(body=None, query=None):
+            return dict(body=body, query=query)
+        self.callback = callback
+
+    def test_no_schema(self):
+        # Use the decorator without the @ syntax:
+        self.callback = rest.schema()(self.callback)
+
+        self.root_app.route('/foo', callback=self.callback)
```

Comment:
I understand why it is there, though. It allows any module to add a route by just decorating a function with `@get`.